README.md

```diff
@@ -2,35 +2,41 @@
 
 
 ## Compatibility
 
-| Module Version / Kubernetes Version | 1.14.X             | 1.15.X             | 1.16.X             | 1.17.X             | 1.18.X             |
-|-------------------------------------|:------------------:|:------------------:|:------------------:|:------------------:|:------------------:|
-| v1.0.0                              |                    |                    |                    |                    |                    |
-| v1.1.0                              |                    |                    |                    |                    |                    |
-| v1.2.0                              | :white_check_mark: | :white_check_mark: | :white_check_mark: |                    |                    |
-| v1.3.0                              | :white_check_mark: | :white_check_mark: | :white_check_mark: |                    |                    |
-| v1.3.1                              | :white_check_mark: | :white_check_mark: | :white_check_mark: |                    |                    |
-| v1.4.0                              |                    |                    | :white_check_mark: | :white_check_mark: | :white_check_mark: |
+| Module Version / Kubernetes Version | 1.14.X             | 1.15.X             | 1.16.X             | 1.17.X             | 1.18.X             | 1.19.X             |
+|-------------------------------------|:------------------:|:------------------:|:------------------:|:------------------:|:------------------:|:------------------:|
+| v1.0.0                              |                    |                    |                    |                    |                    |                    |
+| v1.1.0                              |                    |                    |                    |                    |                    |                    |
+| v1.2.0                              | :white_check_mark: | :white_check_mark: | :white_check_mark: |                    |                    |                    |
+| v1.3.0                              | :white_check_mark: | :white_check_mark: | :white_check_mark: |                    |                    |                    |
+| v1.3.1                              | :white_check_mark: | :white_check_mark: | :white_check_mark: |                    |                    |                    |
+| v1.4.0                              |                    |                    | :white_check_mark: | :white_check_mark: | :white_check_mark: | :white_check_mark: |
+| v1.5.0                              |                    |                    | :white_check_mark: | :white_check_mark: | :white_check_mark: |     :warning:      |
 
 - :white_check_mark: Compatible
 - :warning: Has issues
 - :x: Incompatible
 
+### Warning
+
+- :warning: : module version: `v1.5.0` and Kubernetes Version: `1.19.x`. It works as expected. Marked as warning
+because it is not officially supported by [SIGHUP](https://sighup.io).
```

Comment:
I'm not convinced about this warning... as a minimum I would like to add a "yet" at the end of the phrase, but I think it could be better to say that is a technical-preview or something like that. What do you think?

Reply:
The technical preview warning sounds better.